test/Controller/AboutEarlyCareerControllerTest.php

```diff
@@ -35,6 +35,13 @@ public function it_has_metadata()
         $this->assertSame('The community behind eLife wants to help address some of the pressures on early-career scientists', $crawler->filter('meta[property="og:description"]')->attr('content'));
         $this->assertSame('The community behind eLife wants to help address some of the pressures on early-career scientists', $crawler->filter('meta[name="description"]')->attr('content'));
         $this->assertSame('summary', $crawler->filter('meta[name="twitter:card"]')->attr('content'));
+        $this->assertEmpty($crawler->filter('meta[property="og:image"]'));
```

Comment:
looks like something for a `DublinCore` trait to avoid the same shotgun surgery later on when one of the assertions slightly change

Reply:
The whole metadata tests could do be a rework, will (try and remember to) come back to it.